src/Components/FilterSortBand/FilterSortBand.jsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+
+function FilterSortBand() {
+  return (
+    <div className="flex flex-wrap items-center justify-center mt-7 filtersort-container">
+      <div className="flex flex-wrap justify-start mr-16 bg-gray-200 rounded-lg shadow-xl h-9 filter-container w-72">
+        <div className="flex flex-wrap items-center justify-center w-24 bg-purple-900 rounded-lg filterBy h-9">
+          <img src={require("../../Assets/filter-resto.png")} alt="filter icon" className="w-4 h-4 mr-1 filter-icon" />
```

Comment:
please use import instead of require (require is the old way of doing it) and import it at the beginning of the file and than use it here

Reply:
please also do the same everywhere you use 'require'